kale/loaddata/few_shot.py

```diff
@@ -0,0 +1,124 @@
+"""
+Dataset for N-way-K-shot setting in few-shot problems.
+Author: Wenrui Fan
+Email: winslow.fan@outlook.com
+"""
+
+import os
+from typing import Callable, Optional
+
+import numpy as np
+import torch
+from PIL import Image
+from torch.utils.data import Dataset
+
+
+class NWayKShotDataset(Dataset):
+    """
+    It loads data for N-way K-shot problems in few-shot learning.
```

Comment:
Fix the writing please.

Reply:
Changed to: _It loads data for few-shot learning problems under N-way-K-shot settings._